src/box/sql/prepare.c

```diff
@@ -163,26 +163,23 @@ sql_stmt_compile(const char *zSql, int nBytes, struct Vdbe *pReprepare,
 	return rc;
 }
 
-/*
- * Rerun the compilation of a statement after a schema change.
- */
 int
-sqlReprepare(Vdbe * p)
+sqlReprepare(struct Vdbe *v)
```

Comment:
Why do you rename `p` to `v`? It bloats the diff for no reason. The commit message says nothing about this change.

Reply:
Should I add this to the description? In this patch, I changed the use of the `struct Vdbe`, so I decided to rename variables in some places from `p` (it seems to be short for `pointer`) to the more specialized `v` (short for `vdbe`).

Reply:
I don't think you should change local variable names, because `p` sounds fine to me as a local variable of type `Vdbe`. Moreover, even after this patch is applied, it's still used (e.g. in `sqlVdbeExec`). That is, this change doesn't make the code look any better - it only bloats the diff and makes it more difficult to review.

Reply:
Fixed. Moved all dropped functions in the new commit.

Also, I will try to stick to the "one change = one commit" rule in the future, even with refactoring.